### PR TITLE
Remove archived articles from Inventory at Date export in xls

### PR DESCRIPTION
**Pull request by krunal267**
Task URL:

https://www.druidoo.io/web#id=1442&action=367&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

File: extra_addons/stock_inventory_xlsx/wizard/stock_valuation_history.py [hunk 1]
```diff
@@ -79,7 +82,10 @@ def get_history_datas(self):
             ON stock_move.location_id = source_location.id
         JOIN stock_location dest_location
             ON stock_move.location_dest_id = dest_location.id
+        JOIN product_product product
+            ON stock_move.product_id = product.id
         WHERE  quant.qty > 0
+            AND product.active = 't'
```
Comment (ivantodorovich) [inline extra_addons/stock_inventory_xlsx/wizard/stock_valuation_history.py]:
```suggestion
            AND product.active IS TRUE
```

File: extra_addons/stock_inventory_xlsx/wizard/stock_valuation_history.py [hunk 2]
```diff
@@ -55,7 +55,10 @@ def get_history_datas(self):
             ON stock_move.location_dest_id = dest_location.id
         JOIN stock_location source_location
             ON stock_move.location_id = source_location.id
+        JOIN product_product product
+            ON stock_move.product_id = product.id
         WHERE  quant.qty > 0
+            AND product.active = 't'
```
Comment (ivantodorovich) [inline extra_addons/stock_inventory_xlsx/wizard/stock_valuation_history.py]:
```suggestion
            AND product.active IS TRUE
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: extra_addons/stock_inventory_xlsx/wizard/stock_valuation_history.py (hunk 2 above)
Comment (ivantodorovich) [reply extra_addons/stock_inventory_xlsx/wizard/stock_valuation_history.py]: Just because it's easier to read :)

**Comment by ivantodorovich** [comment]
Can you retarget the PR to github.com/awesomefoodcoops/odoo-production "dev" branch?

**Comment by krunal267** [comment]
Okay, will do the change :)

**Pull request closed without merging**